src/components/ExportEntitiesForm.tsx

```diff
@@ -5,22 +5,58 @@ import { Button, Divider, Link, Typography } from '@material-ui/core';
 import { catalogApiRef } from '@backstage/plugin-catalog-react';
 import { Entity, stringifyEntityRef } from '@backstage/catalog-model';
 import { InfoCard } from '@backstage/core-components';
-import { opslevelApiRef } from '../../api';
+import { opslevelApiRef } from '../api';
 import { useApi } from '@backstage/core-plugin-api';
+import { OpsLevelApi } from '../types/OpsLevelData';
 import { useAsync } from 'react-use';
-import opslevelLogo from '../../images/opslevel-logo.svg';
+import opslevelLogo from '../images/opslevel-logo.svg';
+import { withTheme } from "@material-ui/core/styles";
 
-function useListEntities() {
+function useListEntities(kind: string) {
   const catalogApi = useApi(catalogApiRef);
   return useAsync(async () => {
     return catalogApi.getEntities({
       filter: {
-        kind: 'component',
+        kind,
       },
     });
   });
 }
 
+function useListComponents() {
```

Comment:
Do you think it makes sense to merge those three different entity types and make that dynamic? There's a lot of code duplication going on.

Something like...

```
{
  component: useListEntities('component'),
  user: useListEntities('user'),
  group: useListEntities('group'),
}
```